src/main/java/uk/gov/hmcts/reform/cwrdapi/service/impl/JsrValidatorInitializer.java

```diff
@@ -64,5 +72,19 @@ public List<T> getInvalidJsrRecords(List<T> domains) {
     public Set<ConstraintViolation<T>> getConstraintViolations() {
         return constraintViolations;
     }
+
+    public void validateStaffProfile(T profileRequest) {
+        getInvalidJsrRecords(List.of(profileRequest));
+
+        constraintViolations  = getConstraintViolations();
+        ofNullable(constraintViolations).ifPresent(constraints ->
+                    constraints.forEach(constraintError -> {
+                        String errorMsg =    constraintError.getMessage();
+                        validationServiceFacade.saveStaffAudit(AuditStatus.FAILURE,errorMsg,
+                                null,null);
```

Comment:
Should you not pass the request to the last param of saveStaffAudit instead of null?